docs/using/tutorial/moving-volume-tutorial.rst

```diff
@@ -0,0 +1,192 @@
+The next step is to create two Docker containers on one of the hosts.
+One container has a Python web application and the other has a Redis server, which stores its data on a volume.
+
+Now you can try our simple tutorial: a Python web application and a Redis server.
+Before you begin, you'll need to download our sample ``YAML`` files:
+
+.. container:: hidden
+
+   .. Create the files to be downloaded with ``cURL``, but don't show download links for them.
+
+   :download:`docker-compose.yml`
+   :download:`deployment-node1.yml`
+   :download:`deployment-node2.yml`
+		 
+.. version-code-block:: console
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/docker-compose.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node1.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node2.yml
+
+.. XXX The version-code-block presents a different style to prompts, so this document would benefit from FLOC 2141 (version-prompt)
+
+The :file:`docker-compose.yml` file describes your distributed application:
+
+.. literalinclude:: docker-compose.yml
+   :language: yaml
+
+The :file:`deployment-node1.yml` file describes which containers to deploy, and where:
+
+.. literalinclude:: deployment-node1.yml
+   :language: yaml
+
+#. Use the Flocker CLI to deploy both your web application and server onto one of the virtual machines that you have just created:
+
+   .. prompt:: bash you@laptop:~$
+
+      flocker-deploy 172.16.255.250 deployment-node1.yml docker-compose.yml
+
+#. View both the containers running on node1:
+
+   .. prompt:: bash you@laptop:~$
+   
+      vagrant ssh node1 -c 'docker ps'
+
+.. note:: At this point, you can visit http://172.16.255.250/, where you will see the visit count displayed.
+         
+		 Visit http://172.16.255.251/, where you will see that the count persists because Flocker routes the traffic from either node named in the deployment file to the one that has the application.
+
+Migrating a Container to the Second Host
+========================================
+
+The diagram below illustrates your current server-side Flocker setup:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :alt: In the server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+Now we use the :file:`deployment-node2.yml` file to move the Redis container along with its data volume:
+
+.. literalinclude:: deployment-node2.yml
+   :language: yaml
+
+#. Use the Flocker CLI to migrate one of the containers to the second host:
+
+   .. prompt:: bash you@laptop:~$
+
+      flocker-deploy 172.16.255.250 deployment-node2.yml docker-compose.yml
+
+#. View the Redis container and its volume now running on node2:
```

Comment:
I didn't suggest that because my other suggestion was to remove "host" from this doc. We use "node1", "node2" in Vagrant and "nodes" in the YAML. Host is confusing.